injector/helper.py

```
@@ -124,3 +124,47 @@ def injectLoggingSetup(tree):
+                }
+            }
+        '''
+    """
+
+    validCommentTypes = ["adli_disable_variable","adli_metadata"]   
+
+    if "value" in node._fields and isinstance(node.value, ast.Constant):     
+        comment = node.value.value
+        try:
+            obj = json.loads(comment)
+        except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)